### PR TITLE
[Lite]Bugfix System.loadLibrary exception handle

### PR DESCRIPTION
**Pull request by Dayananda-V**
Exception handle when application couldn't able to load the prebuilt smartreply_jni library.

### CODE REVIEW

File: tensorflow/lite/models/smartreply/demo/app/src/main/java/com/example/android/smartreply/SmartReplyClient.java [hunk 1]
```diff
@@ -53,8 +53,13 @@ public boolean isLoaded() {
   @WorkerThread
   public synchronized void loadModel() {
     if (!isLibraryLoaded) {
-      System.loadLibrary(JNI_LIB);
-      isLibraryLoaded = true;
+      try {
+        System.loadLibrary(JNI_LIB);
+        isLibraryLoaded = true;
+      } catch (Exception e) {
+        Log.e(TAG, "Fail to load prebuilt smartreply_jni lib", e);
```
Comment (karimnosseir) [inline tensorflow/lite/models/smartreply/demo/app/src/main/java/com/example/android/smartreply/SmartReplyClient.java]: nit: Failed to load

**Review by karimnosseir — DISMISSED** [review]
Minor nit. Otherwise LG

Thanks

File: tensorflow/lite/models/smartreply/demo/app/src/main/java/com/example/android/smartreply/SmartReplyClient.java (hunk 1 above)
Comment (Dayananda-V) [reply tensorflow/lite/models/smartreply/demo/app/src/main/java/com/example/android/smartreply/SmartReplyClient.java]: Fixed. Sorry for delay.

**Review by karimnosseir — APPROVED** [review]

**Comment by jdduke** [comment]
Under what conditions do you find that the library fails to load?

**Comment by gbaned** [comment]
@Dayananda-V Did you get a chance to look on jdduke's comments? Please let us know on the update. Thanks!

**Pull request merged**